js/main.js

```diff
@@ -20,9 +22,47 @@ function theSubmit(event) {
   $submitObj.title = titleVal;
   $submitObj.id = data.nextEntryId;
   data.entries.unshift($submitObj);
+  var newListEntries = journalReturn($submitObj);
+  theList.prepend(newListEntries);
   data.nextEntryId++;
   $photoUrl.src = 'images/placeholder-image-square.jpg';
   $formVar.reset();
+  $formVar.className = 'form hidden';
+  location.assign('#entry');
```

Comment:
This is not the best way to do this. You should update the className for the element with `data-view="entries"` to not be hidden, in the same form as you are doing for the `form` on the line above.